rootx/CMakeLists.txt

```diff
@@ -4,16 +4,11 @@
 ############################################################################
 
 if(x11)
-  include_directories(${X11_INCLUDE_DIR})
-  # FIXME: Why are we adding the x11 independent rootx.cxx file?
-  ROOT_EXECUTABLE(root *.cxx  ${CMAKE_SOURCE_DIR}/core/clib/src/strlcpy.c
-                  LIBRARIES ${X11_LIBRARIES} ${X11_Xpm_LIB} ${X11_Xft_LIB} ${X11_Xext_LIB})
-
-  #---CreateRootCommandLineOptions------------------------------------------------------------------
-  generateHeaders(${CMAKE_SOURCE_DIR}/core/base/src/root-argparse.py
-                  ${CMAKE_BINARY_DIR}/include/rootCommandLineOptionsHelp.h root)
+  ROOT_EXECUTABLE(root src/rootx.cxx src/rootxx.cxx ../core/clib/src/strlcpy.c)
+  target_include_directories(root PRIVATE ${X11_INCLUDE_DIR})
```

Comment:
What's the point of declaring properties as `PRIVATE` for executables?

Reply:
I have to put something in there or CMake will complain:
```
CMake Error at rootx/CMakeLists.txt:8 (target_include_directories):
  target_include_directories called with invalid arguments
```